The execution of your program raised a java.lang.RuntimeException exception: Sorry Dave, I cannot let you use forward with an argument in this exercise. Use a loop instead.
 Please fix your code.
   at plm.runtime3.LoopCourse.forward (plm.runtime3.LoopCourse:13)
   at plm.runtime3.LoopCourse.plm$runtime3$LoopCourse$$run$1 (plm.runtime3.LoopCourse:54)
   at plm.runtime3.LoopCourse$$anonfun$circle$1$1.apply$mcVI$sp (plm.runtime3.LoopCourse:59)
   at scala.collection.immutable.Range.foreach$mVc$sp (Range.scala:166)
   at plm.runtime3.LoopCourse.circle$1 (plm.runtime3.LoopCourse:58)
   at plm.runtime3.LoopCourse.run (plm.runtime3.LoopCourse:62)
   at plm.core.lang.JVMCompiledLang.runEntity (null:-1)
   at plm.core.model.lesson.ExerciseRunner$3.run (null:-1)
   at java.lang.Thread.run (Thread.java:745)
